TimeDilationPlugin/TimeDilationPlugin.cs

```diff
@@ -28,13 +29,53 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             Log.Error("TimeDilationPlugin cannot get current sun position, aborting");
```

Comment:
You can move this check so it only triggers when using sun angle based time dilation

Reply:
Moved the if statement to the correct position